webapp/views/App/views/Analysis/Chain/PValueSelector/PValueSelector.js

```diff
@@ -9,7 +9,7 @@ import { FormItem } from '@webapp/components/form/Input'
 import { useI18n } from '@webapp/store/system'
 import { ChainActions, useChain } from '@webapp/store/ui/chain'
 
```

Comment:
what is P?

Reply:
it's the probability value, quite common in analysis

Reply:
oki